classpath/java/lang/Integer.java

```diff
@@ -139,4 +139,14 @@ public static Integer decode(String string) {
     }
     return new Integer(parseInt(string, 10));
   }
+
+  public static int numberOfLeadingZeros(int i) {
```

Comment:
Are you sure this is correct for a signed integer?  I would assume that it needs an unsigned shift right (>>>) at least.